rpc/src/lib.rs

```diff
@@ -138,15 +138,21 @@ fn transaction_build(
 		value: transaction.value,
 		gas_price: transaction.gas_price,
 		gas: transaction.gas_limit,
-		input: Bytes(transaction.input),
+		input: Bytes(transaction.clone().input),
 		creates: status.contract_address,
-		raw: Bytes(vec![]), // TODO
+		raw: Bytes(rlp::encode(&transaction)),
 		public_key: None, // TODO
-		chain_id: None, // TODO
-		standard_v: U256::zero(), // TODO
-		v: U256::zero(), // TODO
-		r: U256::zero(), // TODO
-		s: U256::zero(), // TODO
+		chain_id: {
+			if let Some(chain_id) = transaction.signature.chain_id() {
```

Comment:
`transaction.signature.chain_id().map(U64::from)` would be slightly shorter.

Reply:
That is actually pretty cool :)